pkg/cmd/server/kubernetes/network/options/options.go

```diff
@@ -64,6 +64,12 @@ func Build(options configapi.NodeConfig) (*kubeproxyconfig.KubeProxyConfiguratio
 	masqueradeBit := int32(0)
 	proxyconfig.IPTables.MasqueradeBit = &masqueradeBit
 
+	defaultedProxyConfig, err := proxyOptions.ApplyDefaults(proxyconfig)
```

Comment:
This will end up calling SetDefaults_KubeProxyConfiguration() in pkg/proxy/apis/kubeproxyconfig/v1alpha1/defaults.go, which will force MetricsBindAddress and ResourceContainer to non-empty values, contradicting the values we set earlier in this function. Maybe you want to call this at the start rather than the end?